jetty-io/src/main/java/org/eclipse/jetty/io/DefaultRetainableByteBufferPool.java

```diff
@@ -238,20 +236,20 @@ public static RetainableByteBufferPool findOrAdapt(Container container, ByteBuff
     {
         RetainableByteBufferPool retainableByteBufferPool = container == null ? null : container.getBean(RetainableByteBufferPool.class);
         if (retainableByteBufferPool == null)
-            retainableByteBufferPool = new AdapterMemoryPool(byteBufferPool);
+            retainableByteBufferPool = new ByteBufferToRetainableByteBufferAdapterPool(byteBufferPool);
         return retainableByteBufferPool;
     }
 
     /**
      * An adapter class which exposes a {@link ByteBufferPool} as a
      * {@link RetainableByteBufferPool}.
      */
-    private static class AdapterMemoryPool implements RetainableByteBufferPool
+    private static class ByteBufferToRetainableByteBufferAdapterPool implements RetainableByteBufferPool
```

Comment:
Wow, that's a mouthful, thankfully it's an internal / private class.

Reply:
Any suggestion for a better name?

Reply:
It's private so doesn't really matter as there is good context. `Adaptor` works for me.